src/pcapi/admin/custom_views/offer_view.py

```diff
@@ -121,7 +141,7 @@ class ValidationView(BaseAdminView):
         "metabase": "Metabase",
     }
     column_filters = ["type"]
-    form_columns = ["validation"]
+    column_details_list = ["name"]
```

Comment:
```suggestion
    column_details = ["name"]
```
Proposition. Le mot details implique déjà qu'on va avoir une liste d'éléments